### DISCUSSION TITLE
Page break moves table into new page

### DISCUSSION THREAD

**Discussion by EB-NF** [eclipse-birt/birt#1642] [Q&A]
Hello everyone,

I have an issue with page break when generating PDF using BIRT. 
In case the table does not fit into the first page, it is moved to a second page, and all consecutive page breaks work as expected.

See the example (the table should be in a square marked red):
![paveikslas](https://github.com/eclipse-birt/birt/assets/152991163/63a4e355-5603-443f-aa0c-de280f1849eb)

I've read other posts similar to this issue and tried a number of combinations setting pageBreakBefore, pageBreakAfter and pageBreakInside parameters for the table, row in which the table is placed, and so on, but none gave the expected result.

See the structure of BIRT below:
![paveikslas](https://github.com/eclipse-birt/birt/assets/152991163/4900e09b-5d7b-4970-b51d-c3a0161a9932)

BIRT version: 3.2.23

Please advise.

NOTE: Updating the BIRT version to a newer version is not an option.

**Comment by hvbtup** [comment]
This is probably caused by either pageBreakInside: avoid or pageBreakBefore: always at some point in your report. Yo should carefully examine your report.
